main.py

```diff
@@ -115,7 +115,8 @@ def speak_response(message_sid: str, voice: VoiceType = VoiceType.voice4, cookie
         'TE': 'trailers',
     }
 
-    response = requests.get(f'https://pi.ai/api/chat/voice?messageSid={message_sid}&voice={voice.value}&mode=eager', headers=headers, stream=True)
+    response = requests.get(f'https://pi.ai/api/chat/voice?messageSid={
+                            message_sid}&voice={voice.value[0]}&mode=eager', headers=headers, stream=True)
```

Comment:
I dont think my version of python allows for this syntax of a new line inside the F-string, what version of python are you using?

Reply:
The important bit is that `voice.value` was changed to `voice.value[0]`.  It wouldn't work on 3.12 without it.